Kernel/Storage/NVMe/NVMeQueue.cpp

```diff
@@ -16,7 +16,7 @@ namespace Kernel {
 ErrorOr<NonnullRefPtr<NVMeQueue>> NVMeQueue::try_create(u16 qid, u8 irq, u32 q_depth, OwnPtr<Memory::Region> cq_dma_region, NonnullRefPtrVector<Memory::PhysicalPage> cq_dma_page, OwnPtr<Memory::Region> sq_dma_region, NonnullRefPtrVector<Memory::PhysicalPage> sq_dma_page, Memory::TypedMapping<volatile DoorbellRegister> db_regs)
 {
     auto queue = TRY(adopt_nonnull_ref_or_enomem(new (nothrow) NVMeQueue(qid, irq, q_depth, move(cq_dma_region), cq_dma_page, move(sq_dma_region), sq_dma_page, move(db_regs))));
-    TRY(queue->create());
+    TRY(queue->try_create_dma_buffer());
```

Comment:
Not your fault of course, but would be nice to solve as well: two phase construction is pretty rare in serenity (and rightfully so), so doing this before the constructor is called would be nice.

Reply:
Done :)